backends/graphics/surfacesdl/surfacesdl-graphics.cpp

```diff
@@ -2912,7 +2925,7 @@ void SurfaceSdlGraphicsManager::SDL_UpdateRects(SDL_Surface *screen, int numrect
 	Common::Rect &drawRect = (_overlayVisible) ? _overlayDrawRect : _gameDrawRect;
 	viewport.x = drawRect.left;
 	viewport.y = drawRect.top;
-#if SDL_VERSION_ATLEAST(2, 0, 0)
```

Comment:
Why is this now removed?

Reply:
Because this check is already done above.
This is a separate commit I sneaked in the PR. :)